### ISSUE TITLE
 Undefined index: shortUrl 

### ISSUE THREAD

**Issue by SquanchyMe** [warfare-plugins/social-warfare#832]
See internal #443201
```
PHP Notice: Undefined index: shortUrl in /var/www/.../wp-content/plugins/social-warfare-pro/lib/url-management/SWP_Pro_Rebrandly.php on line 93
[02-Nov-2021 14:23:31 UTC] PHP Notice: Undefined index: id in /var/www/.../wp-content/plugins/social-warfare-pro/lib/url-management/SWP_Pro_Rebrandly.php on line 108
```

**Comment by amyhall-wisersites** [comment]
@SquanchyMe I sent in a pull request for the pro warfare plugin.
Fix - Undefined index: shortUrl #832 #23
Hopefully they will add the fix into the next version.

**Comment by SquanchyMe** [comment]
In testing

**Comment by SquanchyMe** [comment]
No response from customer, marking as fixed in Dev 4.22
